system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -372,6 +374,12 @@ class CLdapConfig : implements ILdapConfig, public CInterface
             m_protocol.append("ldap");
         }
         
+        cfg->getProp(".//@cipherSuite", m_cipherSuite);
+        if(m_cipherSuite.length() == 0)
+        {
+            DBGLOG("Optional LDAPS Cipher Suite not specified");
```

Comment:
Not sure I always want this.  Maybe the other way around ?
("Optional LDAPS cipher suite specified: %s", m_cipherSuite);

Reply:
I would remove it completely as its logged already just above.